src/ProducerSideChannel/include/ProducerSideChannel/ProducerSideChannel.h

```diff
@@ -13,6 +13,8 @@
 
 namespace orbit_producer_side_channel {
 
+constexpr const char* kProducerSideWindowsServerAddress = "localhost:1789";
```

Comment:
The constant that follows is a `std::string_view` and has a comment. Let's be consistent and let's have a comment for this too.

Reply:
I'm not sure about using std::string_view for anything other than function parameters. We can't assign a std::string_view to a std::string for example, which I need here. I don't think there is an advantage to using `std::string_view` over `const char*` here, so I suggest replacing it in the original code.

Reply:
I replaced the original code, please let me know if I overlooked anything. Thanks.

Reply:
I also prefer `const char*` on definition I guess, it's less surprising. But note:

> We can't assign a std::string_view to a std::string for example

You can with `std::string server_address{kProducerSideWindowsServerAddress}`. For reassignment `server_address = std::string{kProducerSideWindowsServerAddress}`. You only can't use `=` implicitly.